www/src/js/templates/snaplist.hbs

```diff
@@ -1,23 +1,29 @@
-{{#if title}}
-<div class="b-headline">
-  <div class="b-layout__container">
-    <h1 class="b-headline__h1">{{ title }}</h1>
-    <div class="b-snaplist__actions">
-      {{#if canSort}}
-      <div class="b-button__group">
-        <div id="sortAlpha" class="b-button b-button_small{{#if isAlpha}} b-button_active{{/if}}">A-Z</div>
-        <div id="sortBytes" class="b-button b-button_small{{#unless isAlpha}} b-button_active{{/unless}}">kB</div>
+<div class="inner-wrapper">
+  {{#if title}}
+  <div class="twelve-col last-col">
```

Comment:
Just a minor suggestion: Maybe it would be possible to add `twelve-col last-col` to `b-headline` element, so it won't be necessary to add additional `div`?

It's also valid for other places in the code. I would suggest that if possible we should add grid or other vanilla classes to existing elements, and only if needed add new elements to HTML.